### PR TITLE
Add ROOGNV

### PR DESCRIPTION
**Pull request by Superfly-Johnson**

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -1804,6 +1804,24 @@ plugins:
         subs: [ '**KFE - Refill Water Bottles.esp**' ]
         condition: 'checksum("KFE - Refill Water Bottles.esp", B85EF60C)'
 
+  - name: 'ROOGNV.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/76452'
```
Comment (pStyl3) [inline masterlist.yaml]: Please add a trailing slash `/` at the end of the link.

**Review by pStyl3 — CHANGES_REQUESTED** [review]

**Review by pStyl3 — APPROVED** [review]

File: masterlist.yaml [hunk 2]
```diff
@@ -1804,6 +1804,24 @@ plugins:
...
+        name: 'ROOG FNV'
+    req:
+      - *JIPLN
+      - 'YUP - Base Game + All DLC.esm'
+      - 'WeaponRepairKitRepaired.esp'
+      - 'WMIMNV.esp'
+    tag:
+      - C.Encounter
+      - Names
+      - Scripts
```
Comment (pStyl3) [inline masterlist.yaml]: Is the `Scripts` Bash Tag needed here? According to WBs [Advanced Readme](https://wrye-bash.github.io/docs/Wrye%20Bash%20Advanced%20Readme.html#ScriptsFO3) it refers to changes to the (SCRI) subrecord..

File: masterlist.yaml [hunk 3]
```diff
@@ -1804,6 +1804,24 @@ plugins:
...
 
+  - name: 'ROOGNV.esp'
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/76452/'
+        name: 'ROOG FNV'
+    req:
+      - *JIPLN
+      - 'YUP - Base Game + All DLC.esm'
+      - 'WeaponRepairKitRepaired.esp'
+      - 'WMIMNV.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: Master files of a plugin do not need to be added to requirements.
LOOT checks the plugin header and shows an error if any master files are missing.

```suggestion
      - 'WeaponRepairKitRepaired.esp'
```

File: masterlist.yaml [hunk 4]
```diff
@@ -1804,6 +1804,22 @@ plugins:
...
+    url:
+      - link: 'https://www.nexusmods.com/newvegas/mods/76452/'
+        name: 'ROOG FNV'
+    req:
+      - *JIPLN
+      - 'WeaponRepairKitRepaired.esp'
+    tag:
+      - C.Encounter
+      - Names
+      - Scripts
```
Comment (MacSplody) [inline masterlist.yaml]: Not required.
```suggestion
```

**Review by MacSplody — APPROVED** [review]

**Pull request merged**